backend/llm/qa.py

```diff
@@ -27,7 +33,26 @@ def get_qa_llm(chat_message: ChatMessage):
     qa = None
     # this overwrites the built-in prompt of the ConversationalRetrievalChain
     ConversationalRetrievalChain.prompts = LANGUAGE_PROMPT
-    if chat_message.model.startswith("gpt"):
+    if openai_api_type and openai_api_type == "azure":
+        qa = ConversationalRetrievalChain.from_llm(
+            AzureChatOpenAI(
+                temperature=chat_message.temperature,
+                model_name=chat_message.model,
+                openai_api_base=openai_api_base,
+                openai_api_version=openai_api_version,
+                deployment_name=openai_api_deployment,
+                openai_api_key=openai_api_key,
+                openai_api_type = openai_api_type,
+                streaming=True,
```

Comment:
Are you sure for this one ?

Reply:
"streaming" is optional, remove it from here.